signalfx-java/src/main/java/com/signalfx/signalflow/ServerSentEventsTransport.java

```diff
@@ -61,6 +63,15 @@ protected ServerSentEventsTransport(String token, final SignalFxEndpoint endpoin
         this.timeout = timeout;
     }
 
+    protected ServerSentEventsTransport(String token, final SignalFxEndpoint endpoint,
+                                        final int apiVersion, final Integer timeout, final Integer maxRetries) {
+        this.token = token;
+        this.endpoint = endpoint;
+        this.path = "/v" + apiVersion + "/signalflow";
+        this.timeout = timeout;
+        this.maxRetries = maxRetries;
+    }
```

Comment:
Best practice here would be for the other constructor to call this one, rather than duplicating the code.